ddtrace/propagation/_database_monitoring.py

```diff
@@ -86,6 +106,14 @@ def _get_dbm_comment(self, db_span):
             DBM_DATABASE_SERVICE_NAME_KEY: service_name_key,
         }
 
+        db_instance = self._find_first_tag(db_span, DBM_PEER_DB_INSTANCE_SOURCE_TAGS)
+        if db_instance:
+            dbm_tags[DBM_PEER_DB_INSTANCE_KEY] = db_instance
+
+        peer_hostname = self._find_first_tag(db_span, DBM_PEER_HOSTNAME_SOURCE_TAGS)
+        if peer_hostname:
+            dbm_tags[DBM_PEER_HOSTNAME_KEY] = peer_hostname
+
         if dbm_config.propagation_mode == "full":
```

Comment:
There's an assumption here that these tags are added during "service mode" - we should validate

Reply:
You mean validate through a test or in the spec? If its in the spec, its already clarified these are used for connection with APM, so they are added when using `service` or `full` mode.